core/relay/scripts/convert-compose.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+comment () {
+    sed -i'.scriptbak' -e "$1"' s/    - /    # - /' "$2"
```

Comment:
This syntax works on both Mac and Linux?

Reply:
Yeah I tested on my mac and my VM (ubuntu based). You also tested this right?